testing/testcafe/tests/form/common.ts

```diff
@@ -0,0 +1,110 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { screenshotTestFn, isMaterial } from '../../helpers/themeUtils';
+import url from '../../helpers/getPageUrl';
+import createWidget from '../../helpers/createWidget';
+import { appendElementTo, insertStylesheetRulesToPage, removeStylesheetRulesFromPage } from '../../helpers/domUtils';
+
+fixture.disablePageReloads`Form`
+  .page(url(__dirname, '../containerQuill.html'));
```

Comment:
check all places where `containerQuill` is used plz

```suggestion
  .page(url(__dirname, '../container.html'));
```

Reply:
We should use '../containerQuill.html' for form testing